packages/content/src/requests/items/transferFileOwnership/TransferFileOwnershipRequestItem.ts

```diff
@@ -0,0 +1,30 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { FileReference, IFileReference } from "@nmshd/transport";
```

Comment:
content may **not** import from transport, you have to move reference and filereference to core-types. Please also move the according tests.